lib/ExpandableListItem/ExpandableListItem.less

```diff
@@ -1,11 +1,12 @@
 /* ExpandableListItem Header*/
 .moon-expandable-list-item-header {
+  padding-right: @moon-spotlight-outset + 30px;
```

Comment:
I haven't looked in detail at this specific case, but in general, we need to move away from CSS rules that are order-dependent. It was actually never a good practice, even before modules, as it would have been far too easy to accidentally break functionality by changing the order of CSS includes. The fix, in the general case, is to write more specific selectors for more specific controls (e.g., moonstone/Item is a more "general" control than moonstone/ExpandablePicker and its header), so that styling provided by the more general control can never override styling provided by the more specific control.